import_streets/src/lib.rs

```diff
@@ -89,9 +89,9 @@ pub enum PrivateOffstreetParking {
     // TODO Based on the number of residents?
 }
 
-/// Create a `StreetNetwork` from an `.osm.xml` file
+/// Create a `StreetNetwork` from the contents of an `.osm.xml` file
 pub fn osm_to_street_network(
-    osm_input_path: String,
+    osm_xml_input: &str,
```

Comment:
This layer shouldn't be IO, because that might happen from a filesystem or network